### PR TITLE
[17.0][ADD] sale_order_invoice_wizard_date

### PR DESCRIPTION
**Pull request by IriaAlonso**
- Add “invoice_date” field when creating multiple order invoices to be included in draft invoices

### CODE REVIEW

**Comment by dalonsod** [comment]
Name for this addon is not accurate enough, better try this one: `sale_order_invoice_wizard_date`.

**Comment by IriaAlonso** [comment]
@dalonsod can u review it?

**Comment by dalonsod** [comment]
OK, I'll test it as soon as possible

File: sale_order_invoice_wizard_date/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models
+class SaleOrder(models.Model):
```
Comment (dalonsod) [inline sale_order_invoice_wizard_date/models/sale_order.py]: Between imports and class definition there should be two blank lines, according usual OCA python guidelines, please add them.

File: sale_order_invoice_wizard_date/wizard/sale_make_invoice_advance.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+class SaleAdvancePaymentInv(models.TransientModel):
```
Comment (dalonsod) [inline sale_order_invoice_wizard_date/wizard/sale_make_invoice_advance.py]: The same as above

**Review by dalonsod — COMMENTED** [review]
Locally tested, :+1: please attend comments anyway. Meanwhile, I'll install it in CmhProd enviroment

**Comment by IriaAlonso** [comment]
@dalonsod changes done

**Review by dalonsod — APPROVED** [review]
Tested by LM in CmhProd environment :+1:

**Pull request merged**